tools/generate_conda_file.py

```diff
@@ -87,13 +87,13 @@
     "pymanopt": "pymanopt==0.2.5",
     "xlearn": "xlearn==0.40a1",
     "transformers": "transformers==2.5.0",
-    "tensorflow": "tensorflow==1.15.2",
+    "tensorflow": "tensorflow>=2.2.2,<2.4",
```

Comment:
ufff, right now we can't upgrade to TF 2, a large part of the code will fail. @dciborow do you now if newer versions of 1.15 are whitelisted https://pypi.org/project/tensorflow/#history?

Reply:
no... Jeremy just looked into this 2.2 is the lowest we can use.

Reply:
Bufff that's really bad news, we need to review a lot of the code.

FYI @Leavingseason

Reply:
re: tensorflow this is my suggestion
1. move functions from reco-utils/common/tf_utils -> reco-utils/recommender/wide_deep/wide_deep_utils (they don't seem to be used outside wide_deep examples)
2. refactor tests so to handle (1)
3. remove use of tensorflow in azureml/wide_deep_training.py (it's only used for printing the version)
4. remove tensorflow as a dependency altogether from core package (which would just be reco-utils/* - recommender module)

Reply:
but.... the tests..... they pass...... can we start off with getting some tests that fail?

Reply:
its not in the PR anymore. Nothing to talk about here....